.ci.yaml

```diff
@@ -3341,6 +3341,7 @@ targets:
       tags: >
         ["devicelab", "hostonly", "mac"]
       task_name: plugin_test
+      presubmit_max_attempts: "2"
```

Comment:
Should we add a bug to track the flakiness fix?

Reply:
I was following from https://chat.google.com/room/AAAAc_4rqiI/-KD7tB8BPSw, where it was suggested to auto retry this target.

Searching from the flaky bugs, I do not see a bug related to `Mac plugin_test`. Also based on the [flaky data](http://data/sites/dash_infra_metrics_datasite/flutter_check_test_flakiness_status_dashboard/?p=BUILDER_NAME:%22Mac%20plugin_test%22&p=LIMIT:100), seems the target is not flaky in recent 100 runs.

I don't think we should mark this target to auto rerun. Closing.